client/src/app/site/motions/modules/motion-detail/components/paragraph-based-amendment/paragraph-based-amendment.component.ts

```diff
@@ -48,7 +50,9 @@ export class ParagraphBasedAmendmentComponent implements OnInit {
 
     public showAllAmendments = false;
 
-    public ngOnInit(): void {}
+    public constructor(protected componentServiceCollector: ComponentServiceCollector) {
+        super(componentServiceCollector);
+    }
```

Comment:
@GabrielInTheWorld this class had no Constructor before. Not sure if this was intended or not.

Reply:
You can, of course, delete these lines, since it is an empty constructor anyway.